homeassistant/components/wemo/entity.py

```diff
@@ -136,6 +138,24 @@ def is_on(self) -> bool:
         """Return true if the state is on. Standby is on."""
         return self._state
 
+    @property
+    def should_poll(self) -> bool:
```

Comment:
Looks like maybe an unrelated change to the wemo component got slipped in accidentally?

Reply:
I don't know from where. I trying to fix it.

Reply:
I'm a bit puzzled.

![image](https://user-images.githubusercontent.com/745542/108607574-7a241700-73c1-11eb-8138-b72c4018554a.png)

This is already in the `dev` branch of yours. How could I fix that?

Reply:
I'll create a new PR.